modules/gitops-release-notes-1-4-5.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-5_{context}"]
+= Release notes for {gitops-title} 1.4.5
+
+[role="_abstract"]
+{gitops-title} 1.4.5 is now available on {product-title} 4.7, 4.8, and 4.9.
+
```

Comment:
If the release is supported on 4.10, then 4.10 needs to mentioned here as well.

Reply:
@wtam2018 are 1.3 and 1.4 supported on and compatible with 4.10? If yes, then we will need to update the first 2 rows of the compatibility matrix here: https://deploy-preview-44078--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes#compatibility-and-support-matrix

Reply:
Post discussions with QE, since docs cover only the major releases in the matrix table, to avoid confusion to the end users, we will add the versions supported in the major releases as per the component matrix maintained by the team.